test/release/examples/primers/interopWithC.chpl

```diff
@@ -0,0 +1,332 @@
+// Interoperability with C
+
+/*
+   By default, Chapel programs are compiled as the main program.  Sometimes the
+   program is intended as a library whose contents can be called from other
+   sources.  This primer will demonstrate how to call Chapel functions from C
+   as well as how to use C symbols in Chapel.
+*/
+
+/*
+   .. _primers-C-interop-Chapel-as-library:
+
+   Chapel as a Library
+   -------------------
+
+   .. _primers-C-interop-functions:
```

Comment:
We might want to rename this (and any references to it) to match the section header below it.